src/cli/msig/index.ts

```diff
@@ -181,6 +187,8 @@ export default async function msig(args: string[]) {
                 await createStakingTmyAccount(options);
             } else if (proposalSubtype === 'contract') {
                 await stakingContractSetup(options);
+            } else if (proposalSubtype === 'buyram') {
+                await buyRam(options);
```

Comment:
why is this part of the staking proposals? this should be at the root levels. aka `yarn run cli msig propose ram buy`